climada/util/api_client.py

```diff
@@ -578,12 +561,8 @@ def get_hazard(self, hazard_type, dump_dir=SYSTEM_DIR, max_datasets=1, **kwargs)
             Directory where the files should be downoladed. Default: SYSTEM_DIR
```

Comment:
Where is `SYSTEM_DIR` defined? Can it be changed by the user? Is this set in the config? This is a very exposed method and should give information that can be immediately digested by the user if possible.

Reply:
Ok. I've added hints in the doc strings.